datatypes-jws/src/commonMain/kotlin/at/asitplus/crypto/datatypes/jws/JsonWebKey.kt

```diff
@@ -27,14 +27,12 @@ data class JsonWebKey(
     val type: JwkType? = null,
     @SerialName("kid")
     val keyId: String? = null,
-    //EC
```

Comment:
Why remove the comment?

Reply:
Because it's not a KDoc comment, so it's not visible from outside the source code, and it provides no added value.

Reply:
since this class mixes up rsa and ec properties as per the spec, I do feel it provides added value to maintainers of the code base